cas-client-core/src/main/java/org/jasig/cas/client/util/MapNamespaceContext.java

```diff
@@ -0,0 +1,62 @@
+package org.jasig.cas.client.util;
+
+import javax.xml.namespace.NamespaceContext;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Namespace context implementation backed by a map of XML prefixes to namespace URIs.
+ *
+ * @author Marvin S. Addison
+ * @since 3.3.4
```

Comment:
Fair enough. Want me to bump the version number in POMs to 3.4.0-SNAPSHOT to be consistent?